### PR TITLE
Feature: Dockerize the application.

### PR DESCRIPTION
**Pull request by h-alice**
As title mentioned, fully dockerized the godbo-backend application.

### CODE REVIEW

File: .env
Comment (NatLee) [inline .env:1]: Is this a placeholder?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# syntax=docker/dockerfile:1
+
+# Godbo app Dockerfile.
+# Use the latest official Golang image
+FROM golang:latest
+
+# Set the Current Working Directory inside the container
+WORKDIR /app
+
+COPY . .
```
Comment (NatLee) [inline Dockerfile]: 用`COPY . .`會把其他`.dockerignore`忘記指定的東西也丟進去image 🥲
是不是可建立一個folder，例如src，然後把需要的檔案丟進去

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+version: '3.8'
+
+services:
+  godbo:
```
Comment (NatLee) [inline docker-compose.yml]: It's good to add `container_name` for more easily recognizing this container.
We can also add `hostname` to make it more cooler. :D

**Review by NatLee — CHANGES_REQUESTED** [review]
Looks good, but there's one question :D
Where is the `README.md`? 🥹

File: .env
Comment (h-alice) [reply .env:1]: Exactly. But currently not used. : D

File: Dockerfile (hunk 1 above)
Comment (h-alice) [reply Dockerfile]: 我覺得可以 這樣東西不會混在一起

File: docker-compose.yml (hunk 2 above)
Comment (NatLee) [reply docker-compose.yml]: Added. 🤩

**Review by NatLee — APPROVED** [review]

**Pull request merged**